portfolio/src/main/webapp/index.html

```diff
@@ -64,7 +64,21 @@ <h3>Interests</h3>
         
         <p>Click here to get a random message:</p>
         <button onclick="addRandomFact()">Fun fact</button>
-        <div id="fact-container"></div>
+        <ul id="fact-container"></ul>
+
+
+        <!-- <p>Click <a href="/hello">here</a> to see content created by server code.</p> -->
```

Comment:
I generally recommend removing commented out code. It's definitely not something that we commit to our projects at work and I think that once you get to a shared/team project it results in quite a bit of clutter in the code.